tests/convert_stata_to_pandas/test_converting.py

```diff
@@ -1,12 +1,12 @@
 from pathlib import Path
 from unittest.mock import MagicMock, patch
 
-from soep_preparation.convert_stata_to_pandas.task import _get_relevant_column_names
+from soep_preparation.utilities.general import get_relevant_column_names
 
 
-@patch("soep_preparation.convert_stata_to_pandas.task.load_module")
+@patch("soep_preparation.utilities.general.load_module")
 @patch("inspect.getsource")
-def test_get_relevant_column_names_with_raw_data_in_docstring(
+def testget_relevant_column_names_with_raw_data_in_docstring(
```

Comment:
Why these name changes? Should always be `test_`, if you need to skip, mark with `@pytest.mark.skip`